lib/DistributionProvider.pm

```diff
@@ -39,6 +39,7 @@ If there is no matched version, then returns Tumbleweed as the default one.
 
 sub provide {
     return Distribution::Sle::AgamaDevel->new() if is_sle() && get_var('VERSION', '') =~ /agama/;
+    return Distribution::Sle::16Latest->new() if is_sle('16+');
```

Comment:
could you move this line to the first? It would make more sense when reading it.

Reply:
updated.